modules/selenium/src/test/java/org/testcontainers/junit/ChromeRecordingWebDriverContainerTest.java

```diff
@@ -17,7 +18,8 @@
     @Rule
     public BrowserWebDriverContainer chromeThatRecordsAllTests = new BrowserWebDriverContainer()
             .withDesiredCapabilities(DesiredCapabilities.chrome())
-            .withRecordingMode(RECORD_ALL, new File("./target/"));
+            .withRecordingMode(RECORD_ALL, new File("./target/"))
+            .withRecordingFileFactory(new DefaultRecordingFileFactory());
```

Comment:
Isn't this already used implicitly? Or did you want to make in explicit in this test?

Reply:
Yeah, it is used implicitly, but wanted to ensure that if you specifically attempted to provide your own factory that it would still succeed.

Reply:
Ok, makes sense.